Harvest.Web/ClientApp/src/App.tsx

```diff
@@ -45,6 +46,7 @@ function App() {
       <ConditionalRoute roles={['FieldManager', 'Supervisor']} path="/quote/create/:projectId" component={QuoteContainer} />
       <Route exact path="/project" component={ProjectListContainer} />
       <Route path="/ticket/create/:projectId" component={TicketContainer} />
+      <Route path="/ticket/allTickets/:projectId" component={AllTickets} />
```

Comment:
just call this `/ticket/all/123`, or /ticket/list/123.  you don't need the duplication on `/ticket/allTickets`.

Reply:
Also this is minor and i'm not even sure why we do it, but top level things we call XYZContainer.  So to keep with our naming scheme we would probably call this `TicketListContainer` or perhaps just `TicketsContainer`